config/routes.rb

```diff
@@ -28,4 +32,7 @@
 
   resources :tenants, as: :tenant, path: '', param: :slug, except: [:destroy]
 
+  get '/404', to: 'errors#not_found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.